### PR TITLE
Working on retrieving data from achaarType collection

### PR DESCRIPTION
**Pull request by CharukaWijethunga**
1. Please check on the achaar.dart , step.dart , and firebase call in firebase_service.dart. 
2. The function is called on task_step_youtube.dart.

### CODE REVIEW

File: untranslated-messages.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (diksha) [inline untranslated-messages.txt:1]: Is this file required.

**Review by diksha — APPROVED** [review]

File: untranslated-messages.txt (hunk 1 above)
Comment (CharukaWijethunga) [reply untranslated-messages.txt:1]: No. Its not required. I just add that to check whether there is untranslated labels or not. I'll remove that.
